tests/resources/config.toml

```diff
@@ -3,8 +3,9 @@ address = "127.0.0.1"
 port = 7878
 
 [Database]
-address = "127.0.0.1"
+type = "postgresql"
+host = "127.0.0.1"
+port = 123
```

Comment:
Postgres's default port is `5432`, we should probably have it set to that.

Reply:
Agreed. Fixed by 596afde.